src/mapillary/interface.py

```diff
@@ -371,10 +401,13 @@ def images_in_bbox(bbox: dict, **filters) -> str:
         ...     organization_id='ORG_ID'
         ... )
     """
-
-    return image.get_images_in_bbox_controller(
-        bbox=bbox, layer="image", zoom=14, filters=filters
-    )
+    if check_bbox_validity(bbox):
+        return image.get_images_in_bbox_controller(
+            bbox=bbox, layer="image", zoom=14, filters=filters
+        )
+    else:
+        print("Please enter the correct bounding box values")
```

Comment:
There should be an exception here. What we can do is creating a class called `InvalidBBoxError` under `models/exceptions.py`, and throw that when reaching this part of the code. :D